package.json

```
@@ -2,6 +2,7 @@
   "name": "@freecodecamp/contribute",
   "type": "module",
   "version": "1.0.0",
+  "packageManager": "pnpm@9.1.2",
```

```suggestion
  "packageManager": "pnpm@9",
```

If we are pinning this, we should only pin the major version. Otherwise we're going to run into nightmares on every patch release (like we did in the recent commit).